integration_tests/test_os.py

```diff
@@ -11,4 +11,9 @@ def test():
     print(read(fd, n))
     close(fd)
 
+    path = "../tmp.txt"
+    fd = open(path, O_WRONLY)
+    write(fd, "Hello World!\n")
```

Comment:
```suggestion
    write(fd, b"Hello World!\n")
```

TODO: fix LPython to make the suggested change work. In fact, I would say first write to `tmp.txt`, and then read from it and then assert if the strings exactly match.

Reply:
For this I followed `visit_ConstantStr()`. I think we need `visit_ConstantBytes()` in `python_ast_to_asr.cpp`. Also, changes in `asr.h` (again, following `StringConstant`).
Is this the correct way @czgdp1807?

Reply:
> Also, changes in `asr.h` (again, following `StringConstant`).

`asr.h` is a generated file. Its generated by the code written in `asdl_cpp.py`.



> I think we need `visit_ConstantBytes()` in `python_ast_to_asr.cpp`

Probably yes.

Reply:
I made those changes. The issue is that the entire `b'Hello World\n'` is treated as a string.
For example, input `char* a_value` to `make_ConstantBytes_t` when printed is same as above.

As a workaround, I have modified `a_value` as:
```
int length = strlen(a_value);
*(a_value+(length-1)) = '\0';
a_value += 2;
```

This works for our requirement (correct string is written to `tmp.txt`), but I am not sure if this is the correct way.